docs/source/install.rst

```diff
@@ -2,19 +2,96 @@
 Installation Guidelines
 =======================
 
-TODO write something
+Install NIX and other dependencies
+==================================
 
-Install Dependencies
-====================
+Because NIXPy is build upon the `NIX library`_ it is necessary to install the NIX library before you can install or build NIXPy on your system.
+The following two sections explain how NIX can be installed on Windows and Linux.
 
-TODO write something
+.. _NIX library: https://github.com/G-Node/nix
 
-Build and Install NIX
-=====================
+Linux
+-----
 
-TODO write something
+If you use the latest Ubuntu LTS you can install NIX from our `PPA`_ on launchpad.
+First you have to add the PPA to your system::
 
-Build and Install NIXPy
-=======================
+    sudo add-apt-repository ppa:gnode/nix
+    sudo apt-get update
+
+Afterwards you can use apt to install the NIX package::
+
+    sudo apt-get install libnix-dev
+
+If you want to use NIX on other distributions you have to compile and install NIX from source.
+
+.. _PPA: https://launchpad.net/~gnode/+archive/ubuntu/nix
```

Comment:
This an external link to our PPA which is used in line 16.  I chose to define all links and references at the end of the section where they are used for the first time. To me this seems to be a good compromise between having them right there where they are used (which may impair the reading flow) and putting them at the end of of the file (where one might not see them).